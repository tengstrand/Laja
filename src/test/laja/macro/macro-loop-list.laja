#macro loop(list)
  #set (separator = "")
  #foreach(element in list)
{separator}{element}##
    #set (separator = ", ")
  #end
#end

#write "target/test-output/macro-loop-list.out"
{loop([1,2,"hi"])}
#end
