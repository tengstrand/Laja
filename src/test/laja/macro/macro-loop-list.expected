1, 2, hi
